app/controllers/api/v2/discovered_hosts_controller.rb

```diff
@@ -145,13 +152,14 @@ def auto_provision_all
               logger.warn "No rule found for host %s" % discovered_host.name
             end
           end
+
           if result
             process_success _("Discovered hosts are provisioning now")
           else
             render_error :custom_error,
               :status => :unprocessable_entity,
               :locals => {
-                :message => _("Errors during auto provisioning: %s") % overall_errors
+                :message => error_message % overall_errors
```

Comment:
Hmm can we really do: `"No discovered hosts to provision" % overall_errors` ?

At least this does not look sane.